rpmbuild/SOURCES/spring-xd-container

```diff
@@ -69,15 +73,32 @@ start() {
             failure
             ;;
     esac
+    CONT_INST=2
+    while [ $RETVAL -le 0 ] && [ $CONT_INST -le $CONTAINERS ]; do
+      echo -n $"Starting $PROG $CONT_INST: "
+      XD_PID=$(runuser $USER -c "$CMD_EXEC $CMD_LINE_OPTS >> $CONTAINER_LOGFILE 2>&1 & echo \${!}");
```

Comment:
didn't see a problem with it, but we could just add a suffix. They do only log to this file during the initial startup before they create their own log with a pid suffix